### PR TITLE
Updates to welcome wagon page for 2023

### PR DESCRIPTION
**Pull request by canncrochet**
General updates for 2023, back in-person, and new location.

----
:books: Documentation preview :books:: https://writethedocs-www--1909.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2023/welcome-wagon.rst [hunk 1]
```diff
@@ -3,5 +3,195 @@
...
+We've gathered info here that will help you navigate the conference like a pro and make you feel more at home. The Welcome Wagon events give new attendees strategies and tips and connect them with people.
+
+Welcome Wagon Tours
+~~~~~~~~~~~~~~~~~~~
+
+Sunday at noon
+Sunday at 2:00pm
+Monday morning at 8:15am
+
+Join us for an informal Introduction to Write the Docs in the Library/Astoria Room. We’ll review information about the conference specifically for first-timers and give everyone a chance to meet someone new. Come on a short tour of the venue so you’ll know where everything is and everything you can take part in.
```
Comment (diggerlilly) [inline docs/conf/portland/2023/welcome-wagon.rst]: Maybe put this before the list of times

File: docs/conf/portland/2023/welcome-wagon.rst [hunk 2]
```diff
@@ -3,5 +3,195 @@
...
+
+-  Portland is a casual-dress town and so is the Write the Docs conference. You'll be meeting business colleagues at this conference, though, so neat and comfortable are good dress guidelines.
+-  If you are going on the `Write the Docs hike on Saturday <https://www.writethedocs.org/conf/portland/2023/outing/>`__, be sure
+   to bring appropriate hiking clothes and shoes. This time of year, the Pacific Northwest tends to be muddy or raining with occasional swaths of blue skies. Layering is usually the way to go.
+
+What will I eat?
+~~~~~~~~~~~~~~~~
+
+-  Snacks are provided for you on conference days.
+-  Coffee, tea, and water are always available in the Crystal Ballroom. Bring a water bottle to make it easier for you to stay hydrated.
```
Comment (diggerlilly) [inline docs/conf/portland/2023/welcome-wagon.rst]: s/Crystal Ballroom/(how are we describing the Show Bar refreshment area?)

File: docs/conf/portland/2023/welcome-wagon.rst [hunk 3]
```diff
@@ -3,5 +3,195 @@
...
+-  Food is available in restaurants in and around Revolution Hall. `Explore Portland's amazing food scene <https://www.writethedocs.org/conf/portland/2023/visiting/>`__. Invite someone you just met to join you! If you are invited to dinner, say yes! Making connections over dinner is a great way to get to know more people.
+
+Where should I sit?
+~~~~~~~~~~~~~~~~~~~
+
+-  The Revolution Hall main stage will have tables and rows of seats behind them.
+-  There's seating in the balcony that you can access from the third floor.
+-  There are no reserved seats; feel free to sit anywhere.
+-  If you can, show up early to the conference each morning to grab a seat at one of the round tables. Introducing yourself to your neighbors is one of the easiest way to meet people.
```
Comment (diggerlilly) [inline docs/conf/portland/2023/welcome-wagon.rst]: just making sure this is still relevant for Rev Hall tables...

File: docs/conf/portland/2023/welcome-wagon.rst [hunk 4]
```diff
@@ -3,5 +3,195 @@
...
+When you get to the conference, come :ref:`say hello <say-hello-2023>`.
+
+We've gathered info here that will help you navigate the conference like a pro and make you feel more at home. The Welcome Wagon events give new attendees strategies and tips and connect them with people.
+
+Welcome Wagon Tours
+~~~~~~~~~~~~~~~~~~~
+
+Sunday at noon
+Sunday at 2:00pm
+Monday morning at 8:15am
```
Comment (plaindocs) [inline docs/conf/portland/2023/welcome-wagon.rst]:
```suggestion
- Sunday at noon
- Sunday at 2:00pm
- Monday morning at 8:15am
```

File: docs/conf/portland/2023/welcome-wagon.rst [hunk 5]
```diff
@@ -3,5 +3,195 @@
...
+-  Join the `Write the Docs Slack <http://slack.writethedocs.org/>`__, and participate in the Welcome Wagon chat room to start making conference connections.
+-  Make a list of two people who are attending with some notes about them and questions for them. Either reach out by email before the conference to set up a meeting onsite or find them at the conference.
+-  Attend the Welcome Wagon tour.
+-  Join in the Saturday hike.
+-  Attend the Sunday writing day and volunteer to help on one of the projects being worked on.
+-  Check out the talk schedule in advance and make note of the talks you don't want to miss.
+-  In the morning, or when you need a break during the day, head down to the Library/Astoria Room to check out the unconference schedule. Make note of any unconference talks you want to attend.
+-  Check out the lightning talks, and get excited about presenting one at next year's conference.
+
+Sample strategy for a second or higher Write the Docs conference
```
Comment (plaindocs) [inline docs/conf/portland/2023/welcome-wagon.rst]:
```suggestion
Sample strategy for a second or subsequent Write the Docs conference
```

**Review by plaindocs — APPROVED** [review]
Looks great overall

File: docs/conf/portland/2023/welcome-wagon.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2023/welcome-wagon.rst]: I reckon not :-p (just to be awkward)

File: docs/conf/portland/2023/welcome-wagon.rst (hunk 3 above)
Comment (ericholscher) [reply docs/conf/portland/2023/welcome-wagon.rst]: Yea, we're working with them to get a few tables, but it won't be as many as in the Crystal for sure.

**Comment by mopdx** [comment]
Looks great!

File: docs/conf/portland/2023/welcome-wagon.rst (hunk 2 above)
Comment (canncrochet) [reply docs/conf/portland/2023/welcome-wagon.rst]: Thank you! According to the notes, it sounds like snacks will be in Library Room. Do you mean the fact that you could buy your own refreshments from Show Bar? I'm not sure when that is open. I thought it was only open during shows.

**Comment by canncrochet** [comment]
I messed up this PR, so closing. Incorporating all feedback here into a new PR I'll make shortly.

**Pull request closed without merging**